src/coreclr/scripts/superpmi-diffs.proj

```diff
@@ -24,13 +24,27 @@
      timeout: %(HelixWorkItem.Timeout)  '"/>
   </Target> -->
 
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(AGENT_OS)' == 'Windows_NT'">
+    <FileSeparatorChar>\</FileSeparatorChar>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(AGENT_OS)' != 'Windows_NT'">
+    <FileSeparatorChar>/</FileSeparatorChar>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(AGENT_OS)' == 'Windows_NT' ">
     <Python>%HELIX_PYTHONPATH%</Python>
     <ProductDirectory>%HELIX_CORRELATION_PAYLOAD%</ProductDirectory>
     <SuperpmiLogsLocation>%HELIX_WORKITEM_UPLOAD_ROOT%</SuperpmiLogsLocation>
     <!-- Workaround until https://github.com/dotnet/arcade/pull/6179 is not available -->
     <HelixResultsDestinationDir>$(BUILD_SOURCESDIRECTORY)\artifacts\helixresults</HelixResultsDestinationDir>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(AGENT_OS)' != 'Windows_NT'">
+    <Python>$HELIX_PYTHONPATH</Python>
+    <ProductDirectory>$HELIX_CORRELATION_PAYLOAD</ProductDirectory>
+    <SuperpmiLogsLocation>$HELIX_WORKITEM_UPLOAD_ROOT</SuperpmiLogsLocation>
+    <!-- Workaround until https://github.com/dotnet/arcade/pull/6179 is not available -->
+    <HelixResultsDestinationDir>$(BUILD_SOURCESDIRECTORY)/artifacts/helixresults</HelixResultsDestinationDir>
```

Comment:
It might be possible, but things in msbuild get evaluated at different times, and I have a very weak understanding of when. So I'm mostly following existing practice (i.e., copy/paste development).

Reply:
You are using `$(FileSeparatorChar)` few lines below to populate the path, so it should work. Just want to simplify the proj file with as little if-else as possible.

Reply:
I'm going to leave it. This case will go away if I ever get around to fixing https://github.com/dotnet/runtime/issues/75169. I'll remove use of FileSeparatorChar in the python paths, since Python can handle `/` properly.